tst/testinstall/perm.tst

```diff
@@ -314,12 +314,38 @@ gap> PermList(Concatenation([1..70000],[70001,1,70003]));
 fail
 
 #
-# LARGEST_MOVED_POINT_PERM
-#
-gap> LARGEST_MOVED_POINT_PERM((2,3));
-3
-gap> LARGEST_MOVED_POINT_PERM((2,70000));
-70000
+# SmallestMovedPoint, LargestMovedPoint, MovedPoints, NrMovedPoints
+#
```

Comment:
Not to say that this makes sense but in the spirit of testing also a bit more unusual inputs, why not add a test for a permutation like (5,7,1) or just (10,1) that is not already sorted?

Reply:
That won't make a difference, because the differences vanishes the moment the permutation object is created, i.e., well before the permutation is passed to those functions.